src/wazuh_db/wdb_agent.c

```diff
@@ -851,25 +752,15 @@ int wdb_update_groups(const char *dirname) {
 
 /* Delete group from belongs table. It opens and closes the DB. Returns 0 on success or -1 on error. */
 int wdb_remove_group_from_belongs_db(const char *name) {
-
-    int result;
-    sqlite3_stmt *stmt;
-
-    if (wdb_open_global() < 0)
-        return -1;
-
-    // Delete from belongs
-    if (wdb_prepare(wdb_global, SQL_DELETE_GROUP_BELONG, -1, &stmt, NULL)) {
-        mdebug1("SQLite: %s", sqlite3_errmsg(wdb_global));
-        return -1;
-    }
-
-    sqlite3_bind_text(stmt, 1, name, -1, NULL);
-
-    result = wdb_step(stmt) == SQLITE_DONE ? (int)sqlite3_changes(wdb_global) : -1;
-    sqlite3_finalize(stmt);
-
-    return result;
+    int result = 0;
+    char wdbquery[OS_BUFFER_SIZE] = "";
+    char wdboutput[OS_BUFFER_SIZE] = "";
+    int wdb_sock = -1;
+
+    snprintf(wdbquery, OS_BUFFER_SIZE, global_db_queries[SQL_DELETE_GROUP_BELONG], name);
+    result = wdbc_query_ex(&wdb_sock, wdbquery, wdboutput, sizeof(wdboutput));
+  
+    return result == 0 ? 0 : -1;
```

Comment:
Log information about the result and make use of `OS_SUCCESS` and `OS_INVALID`

Reply:
Done